Documentation/ceph-common-issues.md

```diff
@@ -12,25 +12,25 @@ If after trying the suggestions found on this page and the problem is not resolv
 
 ## Table of Contents <!-- omit in toc -->
 
-* [Troubleshooting Techniques](#troubleshooting-techniques)
-* [Pod Using Ceph Storage Is Not Running](#pod-using-ceph-storage-is-not-running)
-* [Cluster failing to service requests](#cluster-failing-to-service-requests)
-* [Monitors are the only pods running](#monitors-are-the-only-pods-running)
-* [PVCs stay in pending state](#pvcs-stay-in-pending-state)
-* [OSD pods are failing to start](#osd-pods-are-failing-to-start)
-* [OSD pods are not created on my devices](#osd-pods-are-not-created-on-my-devices)
-* [Node hangs after reboot](#node-hangs-after-reboot)
-* [Rook Agent modprobe exec format error](#rook-agent-modprobe-exec-format-error)
-* [Rook Agent rbd module missing error](#rook-agent-rbd-module-missing-error)
-* [Using multiple shared filesystem (CephFS) is attempted on a kernel version older than 4.7](#using-multiple-shared-filesystem-cephfs-is-attempted-on-a-kernel-version-older-than-47)
-* [Set debug log level for all Ceph daemons](#set-debug-log-level-for-all-ceph-daemons)
-* [Activate log to file for a particular Ceph daemon](#activate-log-to-file-for-a-particular-ceph-daemon)
-* [A worker node using RBD devices hangs up](#a-worker-node-using-rbd-devices-hangs-up)
-* [Too few PGs per OSD warning is shown](#too-few-pgs-per-osd-warning-is-shown)
-* [LVM metadata can be corrupted with OSD on LV-backed PVC](#lvm-metadata-can-be-corrupted-with-osd-on-lv-backed-pvc)
-* [OSD prepare job fails due to low aio-max-nr setting](#osd-prepare-job-fails-due-to-low-aio-max-nr-setting)
-* [Failed to create CRDs](#failed-to-create-crds)
-* [Unexpected partitions created](#unexpected-partitions-created)
+- [Troubleshooting Techniques](#troubleshooting-techniques)
```

Comment:
Did the vscode plugin just change the recommendation on formatting? I thought a while back we changed all the bullets from `-` to `*` and now we're changing them back? What plugin do we all need to have consistency here so we don't change back and forth?

Reply:
Oops yes, missing this one to revert. I'm using https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode.